easybuild/framework/easyblock.py

```diff
@@ -1642,8 +1642,8 @@ def make_module_step(self, fake=False):
 
         txt = ''
         txt += self.make_module_description()
-        txt += self.make_module_extend_modpath()
         txt += self.make_module_dep()
+        txt += self.make_module_extend_modpath()
```

Comment:
Yeah I thought there may well be a reason for this, if so I can always handle it with an `if` statement based on the MNS